README.md

```diff
@@ -144,6 +144,10 @@ Then execute the coding standards checker on Coder itself:
 
     ./vendor/bin/phpcs
 
+Then execute static analysis:
+
+    ./vendor/bin/phpstan
```

Comment:
the correct command is `./vendor/bin/phpstan analyze`

Reply:
Thanks!, the PR has been updated with the proper command. I've used the `analyse` version, as the `analyze` command is the alias for the `analyse` command.